stuff/pipelines/verification_code.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/python3
+
+'''Create verification code to certificates.'''
+
+import logging
+import os
+import sys
+import random
+import MySQLdb
+import MySQLdb.cursors
+
+
+sys.path.insert(
+    0,
+    os.path.join(
+        os.path.dirname(os.path.dirname(os.path.realpath(__file__))), "."))
+
+
+def generate_code(cur: MySQLdb.cursors.BaseCursor) -> str:
+    '''Generate an aleatory code'''
+    diccionary_alfabeth = {"2": 0, "3": 1, "4": 2, "5": 3,
+                           "6": 4, "7": 5, "8": 6, "9": 7,
+                           "C": 8, "F": 9, "G": 10, "H": 11,
+                           "J": 12, "M": 13, "P": 14, "Q": 15,
+                           "R": 16, "V": 17, "W": 18, "X": 19}
+    code_alfabeth = "23456789CFGHJMPQRVWX"
+    condition = True
+    while condition:
+        code_generate = ''.join(random.choices(code_alfabeth, k=9))
```

Comment:
aquí lo más conveniente es trabajar con números hasta el último momento posible, donde se convierten a letras:

```suggestion
        generated_code = random.choices(range(len(_ALPHABET)), k=9)
```

también, se puede evitar el uso de el número mágico `20` si se reemplaza por `len(_ALPHABET)`.